File: ICPC/2015 Divisionals/incomplete/submissions/IncompleteMax.java
import java.util.*;

class IncompleteMax {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int k, d;
		k = sc.nextInt();
		d = sc.nextInt();
		int books = 1;
		int tot = k;
		while (tot <= d) {
			k *= 2;
			tot += k;
			++books;
		}
		System.out.println(books-1);
	}
}

File: ICPC/2015 Divisionals/incomplete/submissions/incomplete_max.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	int k, d;
	cin >> k >> d;
	int books = 1, tot = k;
	while (tot <= d) {
		k *= 2;
		tot += k;
		++books;
	}
	cout << books-1 << endl;
}
